serve.py

```diff
@@ -92,7 +103,7 @@ def render_pid(pid):
     d = pdb[pid]
     return dict(
         weight = 0.0,
-        id = d['_id'],
+        # url = d['url'],
```

Comment:
jak usuwacie kod to po prostu usuńcie, w razie czego można przywrócić. zostawianie zakomentowanego to słaby pomysł, bo im dłużej jest zakomentowany tym mniejszy ma związek z otoczeniem